index.html

```diff
@@ -919,6 +919,93 @@ <h4>
   reader.scan({ recordType: "example.com:sp"});
       </pre>
     </aside>
+    <aside title="Push a smart poster message" class="example">
+      <p>
+        Smart poster defined in NFC Forum Smart Poster Record Type Definition
+        specification. It is an NDEF record that contains an <a>NDEF message</a>
+        as payload, which in turn may contain several records: a URI record that
+        refers to a content, and additional optional records for that content's
+        title, icon, size, type and possible action (e.g. send SMS, make call,
+        launch browser, or save for later, or open for editing, etc).
+      </p>
+      <pre class="highlight">
+        const writer = new NFCWriter();
+        let sp = new NDEFMessage({
+          records: [
+            {
+              recordType: "url",
+              data: "https://my.org/content/19911"
+            },
+            {
+              recordType: "t", // type, a local type to Sp
```

Comment:
maybe explain what it represents? Maybe we should make a non-nominative section quickly explaining what a smart poster may contain?

Reply:
ah you do explain what it means, `type` just made it confusing.

"smart poster type, a local type to Sp"